src/main/java/no/unit/nva/model/contexttypes/LinkedContext.java

```diff
@@ -4,7 +4,22 @@
 
 /**
  * LinkedContexts are contexts that are expressed as a URI, allowing an existing (external) Publication
- * to be used as a PublicationContext. For example, a BookAnthology containing a Chapter.
+ * to be used as a PublicationContext. For example, a BookAnthology containing a Chapter or a Book as part of a series.
+ *
```

Comment:
This does not answer the basic question which is if the container has links to the contained or the contained a link to the container. A sentence of the form "e.g. A Chapter will have a linkedContext to the BookAnthology that contains it" or "A Book Anthology will have a linkedContext to the Chapter". I have understood that it must be the former but one cannot be too clear.

Reply:
Tried to formulate some smart sentences to describe the funtionality